installing/disconnected_install/installing-mirroring-disconnected-v2.adoc

```diff
@@ -0,0 +1,308 @@
+
+You can use the oc-mirror OpenShift CLI (`oc`) plugin to mirror images to a mirror registry in your fully or partially disconnected environments. You must run oc-mirror from a system with internet connectivity in order to download the required images from the official Red Hat registries.
+
+.High level workflow
+
+The following steps outline the high-level workflow on how to use the oc-mirror plugin to mirror images to a mirror registry:
+
+* Create an image set configuration file.
+
+* Mirror the image set to the target mirror registry by using one of the following workflows:
+
+** Mirror an image set directly to the target mirror registry (Mirror-to-Mirror).
+
+** Mirror an image set to disk (Mirror-to-Disk), transfer the tar file to the target environment, then mirror the image set to the target mirror registry (Disk-to-Mirror).
+
+* Configure your cluster to use the resources generated by the oc-mirror.
+
+* Repeat these steps to update your target mirror registry as necessary.
+
+// About V2
+include::modules/oc-mirror-v2-about.adoc[leveloffset=+1]
+
+// oc-mirror compatibility and support
+include::modules/oc-mirror-comp-support.adoc[leveloffset=+2]
+
+
+[id="prerequisites_oc-mirror-v2"]
+== Prerequisites
+
+* You must have a container image registry that supports link:https://docs.docker.com/registry/spec/manifest-v2-2[Docker v2-2] in the location that will host the {product-title} cluster, such as Red Hat Quay.
++
+[NOTE]
+====
+If you use Red Hat Quay, you must use version 3.6 or later with the oc-mirror plugin. If you have an entitlement to Red Hat Quay, see the documentation on deploying Red Hat Quay link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html/deploy_red_hat_quay_for_proof-of-concept_non-production_purposes/[for proof-of-concept purposes] or link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html/deploying_the_red_hat_quay_operator_on_openshift_container_platform/index[by using the Red Hat Quay Operator]. If you need additional assistance selecting and installing a registry, contact your sales representative or Red Hat Support.
+====
+
+* Every machine in the provisioned clusters must have access to the mirror registry. If the registry is unreachable, tasks like installation, updating, or routine operations such as workload relocation may fail. 
+
+Therefore, mirror registries must be operated in a highly available manner, ensuring their availability aligns with the production availability of your {product-title} clusters.
```

Comment:
@subhtk -  should this not be part of the same paragraph above ? (seems like it got misaligned)